.github/workflows/static-analysis.yml

```diff
@@ -22,7 +22,7 @@ jobs:
                 include:
                     - { php-version: "8.0", dependencies: "lowest", pimcore_version: "", phpstan_args: "", experimental: false }
                     - { php-version: "8.1", dependencies: "highest", pimcore_version: "", phpstan_args: "", experimental: false }
-                    #- { php-version: "8.1", dependencies: "highest", pimcore_version: "11.x-dev", phpstan_args: "", experimental: true }
+                    - { php-version: "8.1", dependencies: "highest", pimcore_version: "11.x-dev", phpstan_args: "", experimental: true }
```

Comment:
will require some changes further down too ... 
```
composer require --no-update pimcore/pimcore:"${PIMCORE_VERSION} as 10.99.9"
```
--> should say `11.99.9` I guess